ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -1410,7 +1410,15 @@ namespace OrdinalAxis {
 
             // Check whether the series has at least one point inside the chart
             const hasPointsInside = function (series: Series): boolean {
-                return series.points.some((point): boolean => !!point.isInside);
+                const { min, max } = axis;
+
+                if (defined(min) && defined(max)) {
+                    return series.points.some((point): boolean =>
+                        point.x >= min && point.x <= max
+                    );
+                }
+
+                return false;
```

Comment:
So this change is basically done because `point.isInside` isn't reliable, right? Do we know why `point.isInside` isn't up-to-date? Should we look into a fix that fixes that instead? That may fix other side-effects too, that we are not yet aware of.

Reply:
Yes, the `point.isInside` is failing, because for null points `plotY` property is set to `undefined`.
So it fails right there: https://github.com/highcharts/highcharts/blob/3f75b6145731b0624348c823872215d3e041143e/ts/Core/Series/Series.ts#L3649

As a fix, for `point.isNull === true` we could omit this check, and maybe check only for `plotX` properties?

For example, we could set the compare value of `point.plotY` to 0 if `point.isNull === true`?  Like -> `point.plotY || 0`?

Reply:
Okay, as long as there's a good reason for it, I think we should just leave it like it is. As far as I understand, the `Point.isInside` flag is correct. Null points are not inside, as expected.